docs/dev/libvirt-howto.md

```diff
@@ -286,6 +286,32 @@ kubectl get --all-namespaces pods
 ## Troubleshooting
 If following the above steps hasn't quite worked, please review this section for well known issues.
 
+### console doesn't come up
+In case of libvirt there is no wildcard dns resolution and console is depend on the route which is created by auth operator.
+To make it work we need to use the multi-stage installation and edit the `domain` for ingress.
```

Comment:
nit: No newline for starting new sentences please. Either start a new paragraph or keep the sentence on the same line.

Reply:
@zeenix That what did for minishift also, making every sentence to a new line make it easier to read and navigate. I don't see any issue it here also.

Reply:
If that's true, let's be consistent and start every sentence on a new line? :) This is plain English text so English grammar rules should be followed here as much as they make sense. You break the line based on max characters per line (whatever is decided per project/file) on a word boundary.

As for minishift doing the same, yes I know it has become common practice recently but mostly cause people don't care, not because it helps anything. :-)

Reply:
Starting every sentence on a newline (in a system where newlines on their own don’t matter, as is the case here) is also nice for change control: it means that `diff`s don’t pull in the end of the previous sentence or the start of the next one, and changing one sentence’s length doesn’t cause the rest of the paragraph to reflow and show up in the `diff`...

Reply:
That would apply equally to starting every word on new line, as you often need to fix, add or remove words. :) I hope one thing we can all agree on, is consistency. If you want to start *every* sentence on a new line, maybe we can talk. Otherwise please let's stick to English grammar rules.

Reply:
Yes, I agree we need consistency! IME starting sentences on a new line (and word-wrapping them too) is a nice balance between source readability and long-term maintainability. It’s weird if the source text is also what the user sees, directly, but since most docs nowadays are post-processed and end up as HTML or something like that where newlines are ignored, it’s OK in practice.